tensorflow/compiler/xla/service/gpu/gemm_thunk.cc

```diff
@@ -140,8 +147,7 @@ static bool DoGemmWithAlgorithm(
             *algorithm, output_profile_result)
         .ok();
   }
-
-  if (batch_size != 1) {
+  else if (batch_size != 1) {
```

Comment:
"}" and "else if () {" on the same line

Reply:
why did you change this line?

Reply:
was used for debugging, miss to remove. Done.